falcon/request.py

```diff
@@ -1077,6 +1069,37 @@ def get_header_as_datetime(self, header, required=False, obs_date=False):
                    'Section 7.1.1.1')
             raise errors.HTTPInvalidHeader(msg, header)
 
+    def get_cookie(self, name):
```

Comment:
I have some misgivings about the function name `get_cookie` and its signature.
In Falcon, one can draw parallels between cookie functions and headers functions, they often set or retrieve a **name** :arrow_right: **value** pair, see for instance, [`get_header`](https://falcon.readthedocs.io/en/stable/api/request_and_response.html#falcon.Request.get_header), `set_header`, `set_cookie`, and so on.

However, `get_cookie` would now return a `list`, making it somewhat of an outlier. Could we reconsider alternative names implying a list (such as I remember `get_cookie_values` has been proposed before)?

Reply:
Good point. `get_cookie_values()` might work. Let me see if I can think of any other possible names (suggestions welcome).

Reply:
I don't have any extremely good suggestions unfortunately. I remember I was suggesting *get_cookies*, but that would actually be even more confusing as it would imply all at once. *get_cookie_as_list* would technically fill the bill well, but it sounds somewhat awkward as we don't have other *get_cookie_as_*.